cmd/argocd/commands/logout.go

```diff
@@ -31,6 +38,55 @@ func NewLogoutCommand(globalClientOpts *argocdclient.ClientOptions) *cobra.Comma
 				log.Fatalf("Nothing to logout from")
 			}
 
+			token := localCfg.GetToken(context)
+			if token == "" {
+				log.Fatalf("Error in getting token from context")
+			}
```

Comment:
Getting JWT token for the current context from the file system in local config file.